migrations/1550691913328_schema-Migratefile1.js

```diff
@@ -0,0 +1,39 @@
+exports.up = pgm => {
+
+pgm.createSchema("CrazyBakerSchema");
+
+pgm.createTable("CrazyBakerSchema.Table_Inventory", {
```

Comment:
We do not need column names to have Inv prepended to it, When we retrieve a column we would usually store it in an object that would represent the column 

for example var inventory = call_to_database...

In this case inventory.id should give us an idea of what the id is for. So we can just change these column names to be id, name, price ,type etc. Same applies for table menu

Reply:
new migration file is updated.